rambo/app.py

```diff
@@ -180,14 +198,14 @@ def destroy(ctx=None, vagrant_cwd=None, vagrant_dotfile_path=None):
     click.echo('Temporary files removed')
     click.echo('Destroy complete.')
 
-def export(force=None, resource=None, export_path=None):
+def export(resource=None, export_path=None, force=None):
```

Comment:
The name 'export' probably doesn't make since for this function anymore, we should probably update it some day.

Reply:
Is job is still to move the same chunks of code from where rambo is installed to the cwd. I'm open to alternate names, but it still makes just as much sense to me.

Reply:
I don't follow. But it doesn't matter, we can pull it out later if we want. But it should probably be addressed.